setup.cfg

```diff
@@ -33,7 +33,7 @@ install_requires =
     nbconvert<6
     nglview~=3.0
     numpy~=1.17
-    optimade-client==2022.4.20
+    optimade-client~=2022.9.19
```

Comment:
Note, this can be dangerous, as the versioning of `optimade-client` is not following semantic versioning, but rather, calendar versioning. Hence, major changes may occur from one version to another, without a way to control that through this version number.

I think I would keep the `==` operator here, instead of using the `~=` operator, and simply update the dependency whenever a new release occurs for `optimade-client`, making sure it doesn't break stuff.

Reply:
Thanks! Pinning the version makes sense. Changed.